compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/packaging/repo/PathBaloRepo.java

```diff
@@ -80,14 +80,14 @@ public Patten calculate(PackageID moduleID) {
         
         // if balo file does not exists
         if (Files.notExists(baloPath)) {
-            throw new BLangCompilerException("balo file for dependency [" + dep.getModuleID() + "] does not exists: " +
-                                             dep.getMetadata().getPath().toAbsolutePath().normalize());
+            throw new BLangCompilerException("balo file for module [" + dep.getModuleID() + "] is not found in the " +
+                    "specified path: " + dep.getMetadata().getPath());
         }
     
         // if balo file is not a file
         if (!Files.isRegularFile(baloPath)) {
-            throw new BLangCompilerException("balo file for dependency [" + dep.getModuleID() + "] is not a file: " +
-                                             dep.getMetadata().getPath().toAbsolutePath().normalize());
+            throw new BLangCompilerException("balo file for module [" + dep.getModuleID() + "] is not a regular " +
+                    "file: " + dep.getMetadata().getPath().toAbsolutePath().normalize());
```

Comment:
What's meant by "a regular file"? Wouldn't it be better to simply say that the provided file is not a balo file?

Reply:
+1